docker-compose.yml

```
@@ -1,6 +1,18 @@
 version: '3'
 services:
   # ARM64 dependencies
+  aws_sns_arm64:
```

nit: Seeing as this is using localstack, which can implement multiple AWS services, and that we'll _probably_ want to use it for other integration tests later, maybe we should just call this `aws_arm64` and `aws` respectively? If so, we'd need to update the other references in the docker-compose.yml file

```suggestion
  aws_arm64:
```